### ISSUE TITLE
TypeError: 'successCallback' is an invalid keyword argument for this function

### ISSUE THREAD

**Issue by rajadain** [WikiWatershed/model-my-watershed#2460]
View details in Rollbar: [https://rollbar.com/Azavea/MMW/items/243/](https://rollbar.com/Azavea/MMW/items/243/)

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/views.py", line 452, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/views.py", line 449, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "/opt/app/apps/user/views.py", line 112, in profile
    profile = UserProfile(**params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/base.py", line 480, in __init__
    raise TypeError("'%s' is an invalid keyword argument for this function" % list(kwargs)[0])
TypeError: 'server_errors[]' is an invalid keyword argument for this function
```

**Comment by arottersman** [comment]
We may have to selectively send the profile data. We currently send all model attributes, one of which is a function: 

https://github.com/WikiWatershed/model-my-watershed/blob/5d80d579aadfdefd6265647be1d8a32f4925cf80/src/mmw/js/src/app.js#L153

https://github.com/WikiWatershed/model-my-watershed/blob/5d80d579aadfdefd6265647be1d8a32f4925cf80/src/mmw/js/src/user/views.js#L344

**Comment by jwalgran** [comment]
Do we have a relationship with the person `aufdenkampe+bigcz`? Can we ask them for reproduction steps?

The possible "pollution" of the profile data makes sense, but I am curious about why this did not show up in development.

**Comment by rajadain** [comment]
A lot of variable values during execution can be seen in Rollbar itself: https://rollbar.com/Azavea/MMW/items/243/occurrences/33040555316/

**Comment by jwalgran** [comment]
Indeed. Rollbar points us pretty directly to my code inadvertently including extra state with the POST

<img width="478" alt="screen shot 2017-11-01 at 3 20 48 pm" src="https://user-images.githubusercontent.com/17363/32300817-7582d88c-bf18-11e7-83c0-6b43d72fce1b.png">

**Comment by arottersman** [comment]
I was able to recreate most of this payload but didn't see any errors:

```
successCallback:undefined
success:false
client_errors:
server_errors[]:Server communication error
first_name:alice
last_name:rottersman
organization:azavea
user_type:Unspecified
country:US
postal_code:19147
```

While logged out I went to `/account` and then signed in as a user with `was_skipped` and `is_complete` `false`. Before submitting the modal profile form, I put my browser offline. After the server_error appeared, I put my browser back online and submitted the form successfully.
